app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.update(posts_counter: user.posts.count)
+  end
```

Comment:
- [ ] Great job creating a method that updates the posts counter for a user. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_post_counter` method every time a new post has been saved to database. You can make the `update_post_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :


   ```ruby
     # app/models/post.rb
     after_save :update_post_counter
   
     private
   
     def update_post_counter
       author.increment!(:posts_counter)
     end
   ```

- [ ] Make sure you to use on line 7 `author.increment!` instead of `  user.increment!` since you are using the author_id as the foreign key 👍🏻